backend/models/Presentation.js

```diff
@@ -15,49 +16,39 @@ const Schema = mongoose.Schema;
 // });
 
 // module.exports = mongoose.model("Tweet", tweetSchema);
-
-const slideSchema = new Schema({
+  
+  const presentationSchema = new Schema({
     title: {
       type: String,
       required: true
     },
-    content: {
-      type: String,
-      required: true
-    },
-    image: {
-      type: String
-    }
-    // videoUrl: {
-    //     type: String
-    // }
-  });
-  
-  const presentationSchema = new Schema({
-    title: {
+    category: {
       type: String,
       required: true
     },
-    slides: [slideSchema],
     author: {
       type: mongoose.Schema.Types.ObjectId,
       ref: 'User',
       required: true
     },
-    // videoUrl: {
-    //   type: String
-    // },
-    category: {
-       type: String,
-       required: true
-    },
-    createdAt: {
-      type: Date,
-      default: Date.now
+    slides: {
```

Comment:
for the schema, we need to add the cordinates for the objects within the slide.
im thinking we can set up slides to be:

slides: {
    type: Object,
    required: true 
} 
}, {timestamps: true});
this will allow us to fill be more adaptive during the development phase